### ISSUE TITLE
Not support amo and about: page on 1.3

### ISSUE THREAD

**Issue by ivysrono** [hnakamur/FormatLink-Firefox#4]
https://addons.mozilla.org/en-US/firefox/addon/format-link3/?src=search
about:addons

Click the button, it is empty.
Ok in 1.1

**Comment by hnakamur** [comment]
Could you give me more explanation about "it is empty"?
A screenshot would be nice.

I installed FormatLink 1.3.0 by clicking "Add to Firefox" button on that page.
I needed to reload the existing tabs to use FormatLink.

Maybe you need to reload the existing tabs too.

**Comment by ivysrono** [comment]
1.1：
![default](https://user-images.githubusercontent.com/2649544/27270638-4c240dfc-54f2-11e7-9a83-499f84756e06.png)

1.3：I have restarted Firefox 54
![default](https://user-images.githubusercontent.com/2649544/27270633-3cceee12-54f2-11e7-95bb-db4cd1037a79.png)

**Comment by hnakamur** [comment]
Thank you for your explanation and screenshots.
Could you try to open the settings page of FormatLink and click the "save changes" button
and try it again?

**Comment by ivysrono** [comment]
Still, nothing changed.
I creat one new profile and install only this addon , same problem.

**Comment by hnakamur** [comment]
I reproduced the problem.
It seems this problem occurs on pages at https://addons.mozilla.org/.
This problem does not happen on pages other than https://addons.mozilla.org/.
I'm going to debug this problem.
Thanks.

**Comment by ivysrono** [comment]
This problem is happening on about: pages too.

**Comment by hnakamur** [comment]
It turns out that we cannot get selected text since we cannot use content script in about: pages and addons.mozilla.org pages.
[1310082 - WebExtension content script not working on mozilla.org sites](https://bugzilla.mozilla.org/show_bug.cgi?id=1310082)

I added an error handling for it in [Use page title when getSelection fails · hnakamur/FormatLink-Firefox@d494ccf](https://github.com/hnakamur/FormatLink-Firefox/commit/d494ccf41df63fbd22350804867baafc932ac8b8).
The forthcoming version 1.3.1 will contain this fix.

Also, I found copying to clipboard fails on addons.mozilla.org and about: pages.

In summary, in addons.mozilla.org and about: pages,
* you can use the toolbar button, but `text` value will be always the page URL even if some text is selected.
* you cannot use context menu nor keyboard shortcut to copy the URL.
